Formula/libcdio-paranoia.rb

```diff
@@ -0,0 +1,20 @@
+class LibcdioParanoia < Formula
+  desc "The cd-paranoia audio-cd ripper based on libcdio"
+  homepage "https://www.gnu.org/software/libcdio/"
+  url "https://ftp.gnu.org/gnu/libcdio/libcdio-paranoia-10.2+0.94+2.tar.gz"
+  mirror "https://ftpmirror.gnu.org/libcdio/libcdio-paranoia-10.2+0.94+2.tar.gz"
+  sha256 "d60f82ece97eeb92407a9ee03f3499c8983206672c28ae5e4e22179063c81941"
+
+  depends_on "libcdio" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    system "./configure", "--without-versioned-libs",
```

Comment:
Why `--without-versioned-libs`?

Reply:
Also: does `configure` support `--disable-dependency-tracking` and/or `--disable-silent-rules`? If so, include them.

Reply:
The linker in macOS do not support it. It won't build without it.

Reply:
I added --disable-dependency-tracking and/or --disable-silent-rules per your suggestion. Thank you for the feedback as this is my first formula.